responder/ext/schema/__init__.py

```
@@ -1,14 +1,15 @@
 import os
 from pathlib import Path
 
-import apistar
 import jinja2
 import yaml
 from apispec import APISpec, yaml_utils
 from apispec.ext.marshmallow import MarshmallowPlugin
 
 from responder import status_codes
-from responder.statics import DEFAULT_API_THEME
+from responder.staticfiles import StaticFiles
```

Ruff

`responder.staticfiles.StaticFiles` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)